lib/adal-angular.js

```diff
@@ -125,16 +125,7 @@
                                         // redirect to login requested page
                                         var loginStartPage = _adal._getItem(_adal.CONSTANTS.STORAGE.START_PAGE);
                                         if (loginStartPage) {
-                                            // Check to see if any params were stored
-                                            var paramsJSON = _adal._getItem(_adal.CONSTANTS.STORAGE.START_PAGE_PARAMS);
-                                            if (paramsJSON) {
-                                                // If params were stored redirect to the page and then 
-                                                // initialize the params
-                                                var loginStartPageParams = JSON.parse(paramsJSON);
-                                                $location.url(loginStartPage).search(loginStartPageParams);
-                                            } else {
-                                                $location.url(loginStartPage);
-                                            }
+                                            $location.url(loginStartPage);
                                         }
```

Comment:
We chatted about this, but we should make sure we're saving off and adding back on any hash string.